Makefile

```
@@ -1,23 +1,56 @@
+build-linux-armv6: ## Build for Linux/armv6 to ./out
+	env CGO_ENABLED=0 GOOS=linux GOARCH=arm GOARM=6 go build -ldflags="-X main.version=${BIN_VERSION}" -o ./out/${BIN_NAME}-${BIN_VERSION}-linux-armv6 .
+
+.PHONY: build-darwin-amd64
+build-darwin-amd64: ## Build for macOS/amd64 to ./out
+	env CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build -ldflags="-X main.version=${BIN_VERSION}" -o ./out/${BIN_NAME}-${BIN_VERSION}-darwin-amd64 .
+
+.PHONY: build-darwin-arm64
+build-darwin-arm64: ## Build for macOS/arm64 to ./out
+	env CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -ldflags="-X main.version=${BIN_VERSION}" -o ./out/${BIN_NAME}-${BIN_VERSION}-darwin-arm64 .
+
+.PHONY: package
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)